flask_limiter/extension.py

```diff
@@ -320,7 +320,16 @@ def limiter(self):
         else:
             return self._limiter
 
+
+    def __check_conditional_deductions(self, response):
+        for lim, args in getattr(g, 'conditional_deductions', {}).items():
+            if lim.deduct_when(response):
+                if self.limiter.hit(lim.limit, *args):
+                    g.view_rate_limit = lim
```

Comment:
`g.view_rate_limit = [lim.limit] + args`

Reply:
Thanks! There's a few bugs in that one block actually 🤦